doc/command_ref.rst

```diff
@@ -504,9 +504,9 @@ Check Command
 Check-Update Command
 --------------------
 
-``dnf [options] check-update [--changelogs] [<package-specs>...]``
+``dnf [options] check-update [--changelogs] [<package-name-spec>...]``
```

Comment:
It accepts also a file provide. The question is whatever `check-update` command should accepts same type of argument as the `upgrade` command.